test/e2e/specs/editor/blocks/comments.spec.js

```diff
@@ -9,37 +9,39 @@ const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
  */
 
 test.use( {
-	commentsBlockUtils: async ( { page, admin }, use ) => {
-		await use( new CommentsBlockUtils( { page, admin } ) );
+	commentsBlockUtils: async ( { page, admin, requestUtils }, use ) => {
+		await use( new CommentsBlockUtils( { page, admin, requestUtils } ) );
+	},
+	codeEditorUtils: async ( { page }, use ) => {
+		await use( new CodeEditorUtils( { page } ) );
 	},
 } );
 
 test.describe( 'Comments', () => {
-	let previousPageComments,
-		previousCommentsPerPage,
-		previousDefaultCommentsPage;
+	// Ideally, we'd set options in beforeAll or afterAll. Unfortunately, these
+	// aren't exposed via the REST API, so we have to set them through the
+	// relevant wp-admin screen, which involves page utils; but those are
+	// prohibited from beforeAll/afterAll.
```

Comment:
I think this should still be right before `commentsBlockUtils.setOptions` since it refers to it, no? (I.e. inside `test.beforeEach()`.)

Reply:
We can move it again. I wasn't sure where to place that comment after refactoring those functions, so I left it there.